doc/classes/EditorSettings.xml

```diff
@@ -1216,6 +1216,9 @@
 		<member name="text_editor/appearance/caret/type" type="int" setter="" getter="">
 			The shape of the caret to use in the script editor. [b]Line[/b] displays a vertical line to the left of the current character, whereas [b]Block[/b] displays an outline over the current character.
 		</member>
+		<member name="text_editor/appearance/enable_inline_color_picker" type="bool" setter="" getter="">
+			If [code]true[/code], colored buttons will appear before color constructors. Clicking them will popup a color picker for visually editing the color's parameters.
```

Comment:
A link to the **Color** Variant type is important, I feel.
```suggestion
			If [code]true[/code], displays a colored button before any [Color] constructor in the script editor. Clicking on them allows the color to be modified through a color picker.
```
Perhaps **ColorPicker**, too, but not as much, and it does not parse right for localization.